src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -845,6 +877,24 @@ def load_sids_csmr(key: str, location: str) -> pd.DataFrame:
         raise ValueError(f"Unrecognized key {key}")
 
 
+def load_neonatal_lri_csmr(key: str, location: str) -> pd.DataFrame:
```

Comment:
These two functions appear not to be used any longer and should be deleted.

Reply:
They're used for the CSMR in the affected unmodeled causes.